### PR TITLE
Make strict optional and remove its default usage from the readme example

### PR DESCRIPTION
**Pull request by et13**

### CODE REVIEW

**Comment by et13** [comment]
Could someone take a look at this PR?

File: src/input.ts [hunk 1]
```diff
@@ -123,7 +123,11 @@ function getSeparator(): string {
 }
 
 function getStrict(): boolean {
-    const strict = getRequiredInputString('strict');
+    const strict = getNonRequiredInputString('strict');
```
Comment (MSGaryWang) [inline src/input.ts]: >strict [](start = 10, length = 6)

what we get is string or bool?
"strict": false vs "strict": "false"

Comment (et13) [reply src/input.ts]: Always string - The GitHub API `core.getInput` returns "false" in both cases

Comment (et13) [reply src/input.ts]: Note:  `core.getInput` only returns strings

Looks like GitHub places the input into environment variables which get read back out by `getInput`

https://github.com/actions/toolkit/blob/master/packages/core/src/core.ts
`export function getInput(name: string, options?: InputOptions): string `

**Pull request merged**